presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -195,14 +197,14 @@ public String toText(boolean verbose, int level)
 
     public String toJson()
     {
-        return new JsonRenderer().render(representation);
+        return new JsonRenderer(functionAndTypeManager).render(representation);
     }
 
-    public static String jsonFragmentPlan(PlanNode root, Set<VariableReferenceExpression> variables, FunctionAndTypeManager functionAndTypeManager, Session session)
+    public static String jsonFragmentPlan(PlanNode root, Set<VariableReferenceExpression> variables, FunctionAndTypeManager functionAndTypeManager, StatsAndCosts estimatedStatsAndCosts, Session session)
```

Comment:
Can we reorder the arguments list and move estimatedStatsAndCosts to be after variables? The first 2 arguments and estimatedStatsAndCosts are all inputs/objects that need to be output, functionAndTypeManager and session are assisting objects.

Reply:
ok, reordered the parameters per suggestions